module/batch-action/src/Application/Form/Model/BatchActionFormModel.php

```diff
@@ -27,7 +27,9 @@ class BatchActionFormModel
     public ?BatchActionFilterFormModel $filter = null;
 
     /**
-     * @var mixed
+     * @Assert\Valid()
+     *
+     * @var BatchActionPayloadModel[]|null
      */
-    public $payload = null;
+    public ?array $payload = null;
```

Comment:
Isn't it too specific for our case? Possibly other actions will have completely different payload